docs/api/ShallowWrapper/parent.md

```diff
@@ -2,10 +2,14 @@
 
 Returns a wrapper with the direct parent of the node in the current wrapper.
 
+
```

Comment:
why the extra blank lines?

Reply:
funny... I got the impression that there had to be 3 blank lines between sections - it's pretty common.  I guess not.